compiler/fir/checkers/src/org/jetbrains/kotlin/fir/analysis/diagnostics/FirDefaultErrorMessages.kt

```diff
@@ -411,6 +418,18 @@ class FirDefaultErrorMessages : DefaultErrorMessages.Extension {
             map.put(LEAKED_IN_PLACE_LAMBDA, "Leaked in-place lambda: {2}", SYMBOL)
             map.put(FirErrors.WRONG_IMPLIES_CONDITION, "Wrong implies condition")
 
+            // Nullability
+            map.put(
+                UNSAFE_CALL,
+                "Only safe (?.) or non-null asserted (!!.) calls are allowed on a nullable receiver of type {0}",
+                RENDER_TYPE
+            )
+            //map.put(
+            //    UNSAFE_IMPLICIT_INVOKE_CALL,
```

Comment:
Forgot to mention this. I've tried this by adding a naive check that just checks an `invoke` function call, but then found several cases that require overrides check in general. There are a couple utils that are used at resolution and `fir2ir` regarding `invoke` lookup, and I guess it will require more refactoring, which I guess would be better to be a separate PR/commit.